__snapshots__/DumbContract.spec.ts.js

```diff
@@ -230,6 +239,10 @@ export class DumbContract extends TC.TypeChainContract {
     return TC.promisify(this.rawWeb3Contract.byteArray, []);
   }
 
+  public get returnSmallUint(): Promise<BigNumber> {
```

Comment:
note: these snapshots are used by different target thats why it's BigNumber here